testing/bzlmod/README.md

```diff
@@ -0,0 +1,5 @@
+# bzlmod usage with rules_docker
+
+Porting all of the `rules_docker` in one PR is probably too much, so this is an
```

Comment:
ah, so would need follow up PRs.

Reply:
Yes, language specific docker rules bring a big dependency baggage and I am not yet sure what would be the best way.